Bookish.DataAccess/DataHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using Dapper;
+
+namespace Bookish.DataAccess
+{
+    public class DataHelper
+    {
+        private static readonly IDbConnection DbConnection = new SqlConnection(@"Server = localhost; Database = Bookish; Trusted_Connection = True;");
+
+        public static IEnumerable<T> ExecuteQuery<T>(string query)
```

Comment:
This class should be fully responsible for interacting with the database, therefore it should handle creating the query 🙂